src/main/java/com/blazemeter/taurus/junit/CustomListener.java

```diff
@@ -77,8 +77,10 @@ public void testAssumptionFailure(Failure failure) {
 
     public void testIgnored(Description description) throws Exception {
         log.warning(String.format("ignored %s", description.getDisplayName()));
+        testStarted(description);
```

Comment:
Why we need these calls?

Reply:
Because if test ignored JUnit runner does not call testStarted and testEnded. As result we have not this Sample in result file and Listener has hidden NPE